README.md

```diff
@@ -37,7 +53,7 @@ var mergedNode = new BroccoliMergeTrees(inputNodes, options);
 If this is your `Brocfile.js`:
 
 ```js
-var BroccoliMergeTrees = require('broccoli-merge-trees');
+const BroccoliMergeTrees = require('broccoli-merge-trees');
 
 module.exports = new BroccoliMergeTrees(['public', 'scripts']);
```

Comment:
This snippet is wrong:

* `Brocfile.js` supports exporting a function, just like `ember-cli-build.js`, we should update to
* we should not be recommending that folks use `new` with our main export (use `.MergeTrees` for that)

I believe it should be:

```js
const mergeTrees = require('broccoli-merge-trees');

module.exports = function() {
  return mergeTrees(['public','scripts']);
};
```